docs/sources/reference/commandline/cli.md

```diff
@@ -364,6 +364,20 @@ flag to the Docker daemon as described above.
 Local registries, whose IP address falls in the 127.0.0.0/8 range, are automatically marked as insecure
 as of Docker 1.3.2. It is not recommended to rely on this, as it may change in the future.
 
+### Running a Docker daemon behind a HTTPS_PROXY
+
+When running inside a LAN that uses a `HTTPS` proxy, the Docker Hub certificates
+will be replaced by the proxy's certificates. These need to be added to your Docker
+Host's configuration:
+
+1. Install the `ca-certificates` package for your distribution
+2. Ask your network admin fo the proxy's CA certificate and append them to
+   `/etc/pki/tls/certs/ca-bundle.crt`
+3. Then start your Docker daemon with `HTTPS_PROXY=http://username:password@proxy:port/ docker -d`
```

Comment:
I *think* that depend on how the proxy is configured. Perhaps a short note that this is just for illustrational purposes and that people should ask their network admin for this as well?